model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -342,7 +342,9 @@ case class MethodConfiguration(
                    namespace: String,
                    name: String,
                    rootEntityType: Option[String],
-                   prerequisites: Map[String, AttributeString],
+                   //we used to have prereqs but did nothing with them. so we removed them.
+                   //leaving it as an option means users who are still sending an empty object don't suddenly have their code break.
```

Comment:
is this true? i thought we tolerated extra keys

Reply:
honestly i'm not sure

Reply:
you're right, we do. will fix

Reply:
actually, i'm going to leave it as an option. it's possible that clients have been written that now expect us to return the prerequisites key, so i'm going to accept Nones coming in but put an empty map in the response as it comes out. otherwise it's a breaking API change.